client/src/components/EmployeeRow/EmployeeRow.tsx

```diff
@@ -18,7 +18,7 @@ const EmployeeRow = ({ employee }: Props) => {
         <div
           className="employee-row-profile-button"
           onClick={() => {
-            navigate(`/profile/${employee.email.split("@")[0]}`);
+            navigate(`/profile/${employee.id}`);
```

Comment:
why

Reply:
It didn't use to work the way it was implemented. I tried it with ID as a route and it worked well.